src/riskmatrix/models/risk_assessment.py

```diff
@@ -146,3 +146,7 @@ def __acl__(self) -> list['ACL']:
         return [
             (Allow, f'org_{self.risk.organization_id}', ['view']),
         ]
+
+
+class RiskMatrixAssessment(RiskAssessment):
+    nr: ClassVar[int]
```

Comment:
```suggestion
    nr: int
```
Based on how this is used, this shouldn't be a `ClassVar`.